src/protocol/invite.rs

```diff
@@ -32,7 +32,7 @@ import_exception!(parsec.api.protocol, ProtocolError);
 
 #[pyclass]
 #[derive(Clone)]
-pub(crate) struct InvitationType(pub libparsec::types::InvitationType);
+pub(crate) struct InvitationType(pub libparsec::types::invite::InvitationType);
```

Comment:
why this change ?

We precisely wan simple import path, this is making things more complex !